pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -2197,20 +2200,31 @@ func checkIoBundle(ctx *domainContext, ib *types.IoBundle) error {
 	return nil
 }
 
+// Move the USB controllers to/from pciback based on usbAccess
+// Also enable/disable usbhid and related mouse/keyboard based on that
+// XXX should we have a separate knob for HID and for usb-storage?
```

Comment:
I'd rather not introduce a separate knob just yet -- lets see how it evolves with the change that I'll be working on next week first.

Reply:
Yeah it isn't clear to me yet (because I have spent sufficient time thinking about it) how to best handle usb storage so we can have the benefit of the override.json, yet be able to disable usb-storage once you don't need it). That is orthogal to whether you want to be able to do on-site debugging using a keyboard/mouse.

Reply:
Anyhow, not planning on doing this XXX as part of this PR, but need to figure out the direction at some point.